commands/shop.js

```diff
@@ -1,9 +1,9 @@
 exports.type = "sburb";
 exports.desc = "Buy items";
-exports.use = `">shop" lets you see what a merchant at your location has in stock.
-">shop [number]" will buy the chosen item if you have the boondollars for it.
-">shop price [number]" will tell you the price of the chosen item.
-">shop inspect [number]" will allow you to inspect the chosen item more thoroughly.`;
+exports.use = `";shop" lets you see what a merchant at your location has in stock.
+";shop [number]" will buy the chosen item if you have the boondollars for it.
+";shop price [number]" will tell you the price of the chosen item.
+";shop inspect [number]" will allow you to inspect the chosen item more thoroughly.`;
```

Comment:
the help command should automatically filter > into your bot's proper prefix, so there shouldn't be a need to do this